src/main/java/edu/kis/powp/jobs2d/drivers/macro/MacroDriverDecorator.java

```diff
@@ -10,22 +10,18 @@
 import java.util.List;
 
 
-public class MacroAdapter implements Job2dDriver {
+public class MacroDriverDecorator implements Job2dDriver {
     private List<DriverCommand> driverCommandList;
-
     private Job2dDriver driver;
 
-    public MacroAdapter() {
+    public MacroDriverDecorator() {
         this.driverCommandList = new ArrayList<>();
     }
 
-    public MacroAdapter(List<DriverCommand> driverCommandList) {
-        this.driverCommandList = driverCommandList;
+    public void setDriver() {
+        this.driver = DriverFeature.getDriverManager().getCurrentDriver();
```

Comment:
Unnecessary dependency to DriverManager and DriverFeature. Driver should be injected (as a constructor param).